pkg/ciliumenvoyconfig/cec_resource_parser.go

```diff
@@ -623,7 +622,8 @@ func validateAddress(namespace, name string, address *envoy_config_core.Address,
 
 // validateEDSEndpoints checks internal listener references, if any
 func validateEDSEndpoints(namespace, name string, eds *envoy_config_endpoint.ClusterLoadAssignment,
-	listeners []*envoy_config_listener.Listener) error {
+	listeners []*envoy_config_listener.Listener,
+) error {
```

Comment:
is it intentional ?

Reply:
yes, intentional as per `gofumpt`  :smiley: if it's ok :pray: